program/src/lib.rs

```diff
@@ -66,18 +65,11 @@ pub mod reward_center {
         offers::create::handler(ctx, create_offer_params)
     }
 
-    pub fn update_offer(
-        ctx: Context<UpdateOffer>,
-        update_offer_params: UpdateOfferParams,
+    pub fn close_offer(
+        ctx: Context<CloseOffer>,
+        close_offer_params: CloseOfferParams,
     ) -> Result<()> {
-        offers::update::handler(ctx, update_offer_params)
-    }
-
-    pub fn cancel_offer(
-        ctx: Context<CancelOffer>,
-        cancel_offer_params: CancelOfferParams,
-    ) -> Result<()> {
-        offers::cancel::handler(ctx, cancel_offer_params)
+        offers::close::handler(ctx, close_offer_params)
```

Comment:
Renamed to close_* to signal better that the accounts are being closed and removed the update offer instruction. Instead execute close offer and then create offer.